tox.ini

```diff
@@ -2,16 +2,18 @@
 min_version = 4.0
 
 envlist =
-    python{3.8,3.9,3.10}-django{3.2}-wagtail{4.1,5.0,5.1}
-    python{3.9,3.10,3.11}-django{4.1}-wagtail{4.1,5.0,5.1}
-    python{3.10,3.11}-django{4.2}-wagtail{5.0,5.1}
+    python{3.8,3.9,3.10}-django{3.2}-wagtail{4.1,5.0,5.1,5.2}
```

Comment:
can you please drop 5.0 from the test matrix and deps. it is officially EOL - https://endoflife.date/wagtail

Reply:
Hi @zerolab ,
I removed it here and updated the one on README.md:
https://github.com/torchbox/wagtail-footnotes/pull/62/commits/6e94fa53269af52a1e27ac767eb839efa639a4e8

Thanks for checking!